src/main/java/core/basesyntax/service/OperationStrategy.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface OperationStrategy {
+    void useOperation(FruitTransaction.Operation operation);
```

Comment:
```suggestion
    OperationExecutor get(FruitTransaction.Operation operation);
```
this method should return an instance of executor